lib/ContentTypeParser.js

```diff
@@ -65,7 +77,8 @@ ContentTypeParser.prototype.getParser = function (contentType) {
 ContentTypeParser.prototype.run = function (contentType, handler, request, reply) {
   var parser = this.cache.get(contentType) || this.getParser(contentType)
   if (parser === undefined) {
-    reply.code(415).send(new Error('Unsupported Media Type: ' + contentType))
+    var err = new FST_ERR_CTP_INVALID_MEDIA_TYPE(`Unsupported Media Type: ${contentType}`)
```

Comment:
I think we should move the message to the `errors` file  as well.

Reply:
yes, would produce more readable code.